util/find_missing_translations.js

```diff
@@ -87,10 +87,16 @@ const parseJavascriptFile = (file, locales) => {
   let keys = [];
   let openMatch = null;
 
-  const openObjRe = new RegExp('(\\s*)(.*{)\\s*');
+  const openObjRe = new RegExp('(\\s*)(.*{)\\s*$');
```

Comment:
Sorry to be slow, but how does this fix the variable issue? I was expecting a change to `keyRe`?

Reply:
This was matching lines like `en: 'Hello ${name}'` based off the `{`, since `\\s*` can match 0 results as well.

Reply:
Oho! Thanks for the explanation.